frontend/test/metabase-visual/dashboard/parameters-widget.cy.spec.js

```diff
@@ -80,41 +84,44 @@ describe(`visual tests > dashboard > parameters widget`, () => {
       });
     });
 
-    describe(`mobile`, () => {
-      it("is sticky in view mode", () => {
-        cy.viewport(MOBILE_WIDTH, 667);
-
-        cy.get("main")
-          .scrollTo(0, 264)
-          .then(() => {
-            cy.createPercySnapshot(null, { widths: [MOBILE_WIDTH] });
-          });
-
-        cy.findByTestId("dashboard-parameters-widget-container").should(
-          "have.css",
-          "position",
-          "fixed",
-        );
-      });
-
-      it("is sticky in edit mode", () => {
-        cy.viewport(MOBILE_WIDTH, 667);
+    describe(
+      "mobile",
+      {
```

Comment:
This is awesome. I never knew we could do this 🎉 

Ref: https://docs.cypress.io/guides/references/configuration#Test-specific-Configuration